types/react/index.d.ts

```diff
@@ -3007,4 +3013,33 @@ declare global {
             view: React.SVGProps<SVGViewElement>;
         }
     }
+
+    type DeepReadonly<T> =
```

Comment:
For anything that's not public API, can you move these outside the `React` namespace? So that it's inaccessible to consumers.

Reply:
Sure. I haven't noticed it's inside `declare Global`. I'll move it to be above it.